cartographer/mapping/trajectory_builder_interface.h

```diff
@@ -70,6 +71,8 @@ class TrajectoryBuilderInterface {
   virtual void AddSensorData(
       const std::string& sensor_id,
       const sensor::FixedFramePoseData& fixed_frame_pose) = 0;
+  virtual void AddPoseGraphData(
```

Comment:
Perhaps add a comment that it allows passing local slam results for using global SLAM only?

Reply:
Done.